miprometheus/problems/question_context_to_class/babiqa_dataset_single_question.py

```diff
@@ -0,0 +1,514 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) IBM Corporation 2018
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, e:wqither express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""bAbiQA.py: contains code for loading the babi dataset (based on the parsing used in torchtext)"""
+__author__= "Vincent Albouy, Ryan.L McAvoy"
+
+import torch
+from miprometheus.utils.problems_utils.language import Language
+import torch.utils.data
+from tqdm import tqdm
+import os
+from miprometheus.utils.app_state import AppState
+from miprometheus.problems.seq_to_seq.seq_to_seq_problem import SeqToSeqProblem
+from miprometheus.utils.loss.masked_cross_entropy_loss import MaskedCrossEntropyLoss
+
+
+
+class BABI(SeqToSeqProblem):
```

Comment:
The name of dataset is bAbI (AI with capital letters), and as this is a version with single question in a story (if I understand it correctly), I suggest renaming it to something that will reflect that fact, e.g. 
bAbIQASingleQuestion

Reply:
Ok done